LICENSE

```diff
@@ -299,3 +299,13 @@ This product includes code from Delta Lake.
 Copyright: 2020 The Delta Lake Project Authors.
 Home page: https://delta.io/
 License: https://www.apache.org/licenses/LICENSE-2.0
+
+--------------------------------------------------------------------------------
+
+This product includes code from Apache Commons.
+
+* Core ArrayUtil.
+
+Copyright: 2020 The Apache Software Foundation
+Home page: https://commons.apache.org/
+License: https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
This needs to be added to all of the runtime LICENSE files as well, like spark3-runtime/LICENSE. Can you add it to all of the runtime license files, please?

Reply:
Sure.